core/consensus/aura/src/lib.rs

```diff
@@ -328,15 +327,37 @@ fn check_header<B: Block>(slot_now: u64, mut header: B::Header, hash: B::Hash, a
 	}
 }
 
+/// Extra verification for Aura blocks.
+pub trait ExtraVerification<B: Block>: Send + Sync {
+	/// Future that resolves when the block is verified or not.
```

Comment:
```suggestion
	/// Future that resolves when the block is verified or fails with an error if not.
```

Reply:
to be more explicit that we aren't actually checking the result at all, so returning a `false` in the future would still be considered as this having been`verified`.

Reply:
there is no `bool` here but i have made the change as suggested. thanks